pii_recognition/evaluation/pipeline_test.py

```diff
@@ -26,9 +27,23 @@ def mock_registry():
 
 @patch("pii_recognition.evaluation.pipeline.recogniser_registry", new=mock_registry())
 def test_get_recogniser():
-    actual = get_recogniser("RegistryNoConfig")["recogniser"]
+    tokeniser_setup = {"name": "RegistryNoConfig"}
+    actual = get_recogniser(tokeniser_setup)["recogniser"]  # it's in meta
+    assert isinstance(actual, RegistryNoConfig)
+
+    tokeniser_setup = {"name": "RegistryWithConfig", "config": {"param_a": "value_a"}}
+    actual = get_recogniser(tokeniser_setup)["recogniser"]  # it's in meta
+    assert isinstance(actual, RegistryWithConfig)
+    assert actual.param_a == "value_a"
+
+
+@patch("pii_recognition.evaluation.pipeline.tokeniser_registry", new=mock_registry())
+def test_get_tokeniser():
+    tokeniser_setup = {"name": "RegistryNoConfig"}
+    actual = get_tokeniser(tokeniser_setup)
     assert isinstance(actual, RegistryNoConfig)
 
-    actual = get_recogniser("RegistryWithConfig", {"config_a": "attr_a"})["recogniser"]
+    tokeniser_setup = {"name": "RegistryWithConfig", "config": {"param_a": "value_a"}}
+    actual = get_tokeniser(tokeniser_setup)
```

Comment:
`mypy` would raise error on this
`Dict entry 1 has incompatible type "str": "Dict[str, str]"; expected "str": "str"`

Reply:
Reassignment causes the issue.